### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by ColdBlacksmith** [FTBTeam/FTB-Interactions#1356]
## Modpack Version: 
2.0.9

## Bug Report:
Error during startup that prevents Minecraft from launching, lists Modular Machinery as potential cause

## Expected Behavior/Suggestions:
Minecraft is expected to load, instead it produces an error

## Is It Repeatable? Steps to Reproduce: 
Yes,
1. Hit Play in FTB-app
2.HIt launch in Minecraft launcher
3. Hit ok when Minecraft warns that it is a modified version of Minecraft

## Link to Log or Crash File Paste**: 
It generates a crash file every time, however when I try to open it the get link button changes to say failed.
The most recent link shows: crash-2021-03-27_19.20.34-client.txt 
![Screenshot (51)](https://user-images.githubusercontent.com/81494426/112737819-b3f1bb80-8f33-11eb-9c68-a8d6bd100b49.png)

## Mod/s Affected: 
Modular Machinery

**Comment by michael-zalewski** [comment]
Were you by any chance using Logistocs Pipes Fluid Satellite pipe with your modular machine?

**Comment by ColdBlacksmith** [comment]
Nope, I hadn't even created a single world yet. This was from my first time attempting to play Interactions and the error still persists
